doc/ampi/03-using.rst

```diff
@@ -520,6 +574,7 @@ different schemes.
    Privatization Scheme Linux Mac OS BG/Q Windows x86 x86_64 PPC   ARM7
    ==================== ===== ====== ==== ======= === ====== ===== =====
    Manual Code Editing  Yes   Yes    Yes  Yes     Yes Yes    Yes   Yes
+   PIE-Globals          Yes   No     No   No      Yes Yes    Yes   Yes
```

Comment:
Maybe synchronize the names here with the names in the section headers

Reply:
Done